packages/cli/src/cli/cmd/cleanup.ts

```diff
@@ -11,10 +11,12 @@ export default new Command()
   .command("cleanup")
   .description("Remove keys from target files that do not exist in the source file")
   .helpOption("-h, --help", "Show help")
-  .option("--locale <locale>", "Specific locale to cleanup")
-  .option("--bucket <bucket>", "Specific bucket to cleanup")
-  .option("--dry-run", "Show what would be removed without making changes")
-  .option("--verbose", "Show verbose output")
+  .option("--locale <locale>", "Clean up only the specified target locale.If not provided, processes all target locales.")
+  .option("--bucket <bucket>", " Clean up only the specified bucket type.If not provided, processes all buckets.")
```

Comment:
```suggestion
  .option("--locale <locale>", "Clean up only the specified target locale")
  .option("--bucket <bucket>", " Clean up only the specified bucket type")
```

Lets not make this too verbose.

Reply:
ok changed